build/BuildInfo.targets

```diff
@@ -0,0 +1,40 @@
+<Project ToolsVersion="15.0">
+  <Target Name="WriteBuildInfoProps"
```

Comment:
I cannot depend on HostInfo.targets because I cannot guarantee that I will read the properties. HostInfo.targets generates HostInfo.props. Consider the two scenarios:
1. HostInfo.props does not exist. Then HostInfo.targets generates it and I have the desired properties because I called GetCurrentRuntimeInformation.

2. HostInfo.props already exists. Then HostInfo.targets does not generate it and I don't have the properties.

Note that I also cannot depend on loading HostInfo.props because it may or may not exist, and I cannot conditionally load a props file from within a target (to make sure I generated it first).